src/Parser/Rogue/Subtlety/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,45 @@
+import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+import SPELLS from 'common/SPELLS';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+
+    static BASE_GCD = 1000;
+    static MINIMUM_GCD = 1000;
+    
+    static ABILITIES_ON_GCD = [
+        SPELLS.BACKSTAB.id,
+        SPELLS.GLOOMBLADE_TALENT.id,
+        SPELLS.SHADOWSTRIKE.id,
+        SPELLS.SHURIKEN_STORM.id,
+        SPELLS.SHURIKEN_TOSS.id,
+        SPELLS.GOREMAWS_BITE.id,
+
+        SPELLS.DEATH_FROM_ABOVE_TALENT.id,
+
+        SPELLS.EVISCERATE.id,
+        SPELLS.NIGHTBLADE.id,
+        SPELLS.KIDNEY_SHOT.id,
+
+        SPELLS.CRIMSON_VAIL.id,
+        SPELLS.FEINT.id,
+    ]
+
+    static STATIC_GCD_ABILITIES = {
+        [SPELLS.DEATH_FROM_ABOVE_TALENT.id]: 2000,
```

Comment:
A static 2 second GCD? Blizz pls.

Reply:
You're jumping in the air and such anyways